dask/dataframe/io/tests/test_demo.py

```diff
@@ -81,7 +84,8 @@ def test_make_timeseries_no_args():
 
 
 def test_make_timeseries_blockwise():
-    df = dd.demo.make_timeseries()
+    with dask.config.set({"dataframe.convert_string": False}):
+        df = dd.demo.make_timeseries()
```

Comment:
Hrm, why this change?

Reply:
This was actually not needed, I removed it.